packages/spor-theme-react/src/components/tabs.ts

```diff
@@ -83,11 +83,33 @@ const getTabColorSchemeSelectedProps = (props: StyleFunctionProps) => {
         _focus: {
           boxShadow: `inset 0 0 0 2px ${props.theme.colors.alias.greenHaze}`,
         },
+        "&:focus:not(:focus-visible)": {
+          boxShadow: "none",
+        },
+        _focusVisible: {
+          boxShadow: `inset 0 0 0 2px ${props.theme.colors.alias.greenHaze}`,
+        },
+        _hover: {
+          backgroundColor: "alias.white",
+          color: "alias.darkTeal",
+        },
+        _active: {
+          backgroundColor: "alias.white",
+          color: "alias.darkTeal",
+        },
```

Comment:
Disse to er lagt til for å ikke ha noen hover eller active styles på den valgte taben